### PR TITLE
amass-track: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/amass-track.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# amass track
+
+> Seguimiento de las diferencias entre enumeraciones del mismo dominio.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-track-subcommand>.
+
+- Muestra la diferencia entre las dos últimas enumeraciones del dominio especificado:
+
+`amass track -dir {{ruta/a/directorio_de_base_de_datos}} -d {{nombre_dominio}} -last 2`
```
Comment (EmilySeville7cfg) [inline pages.es/common/amass-track.md:8]:
```suggestion
- Show the difference between the last enumerations of a specific domain::

`amass track -dir {{ruta/a/directorio_de_base_de_datos}} -d {{nombre_dominio}} -last {{1..infinity}}`
```

Optional fix: `Show the difference between the last enumerations of a specific domain:` can be used as a translation source, `specific` may sound better.
Mandatory fix: `-last` option doesn't accept just `2` integer literal, placeholder is required, range syntax can be used.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

File: pages.es/common/amass-track.md (hunk 1 above)
Comment (kbdharun) [reply pages.es/common/amass-track.md:8]: Based on the above suggestion feel free to translate it.

Comment (EmilySeville7cfg) [reply pages.es/common/amass-track.md:8]: Done. :)

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages.es/common/amass-track.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# amass track
+
+> Seguimiento de las diferencias entre enumeraciones del mismo dominio.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-track-subcommand>.
+
+- Mostrar la diferencia entre las últimas enumeraciones de un dominio específico:
```
Comment (navarroaxel) [inline pages.es/common/amass-track.md:6]:
```suggestion
- Muestra la diferencia entre las últimas enumeraciones de un dominio específico:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**